rskj-core/src/main/java/co/rsk/rpc/ModuleDescription.java

```diff
@@ -80,6 +86,24 @@ public boolean methodIsInModule(String methodName) {
         return true;
     }
 
+    public long getTimeout() {
+        return timeout;
+    }
+
+    public long getTimeout(String methodName, long defaultTimeout) {
+        if (methodName.isEmpty()) {
```

Comment:
seems `methodName ` cannot be `null` here? If you really wanna validate this arg you could throw `NPE`/`IllegalArgumentException`. Also method is not used publicly, so can it be `private`?

Reply:
Addressed the exception handling, but this is is being accessed from `JsonRpcCustomServer`